app/config/AppConfig.scala

```diff
@@ -106,7 +107,14 @@ class AppConfig @Inject()(
     runModeConfiguration
       .getOptional[String]("list-of-available-journeys")
       .map(_.split(","))
-      .getOrElse(Array(Choice.AllowedChoiceValues.SupplementaryDec))
+      .getOrElse(Array(Choice.AllowedChoiceValues.Submissions))
```

Comment:
I'm not sure that Submissions is a journey. This should be always on Choice screen